.github/workflows/deployment.yml

```diff
@@ -361,23 +374,33 @@ jobs:
         if: (steps.checksecrets.outputs.secretspresent == 'YES')
         shell: bash
         run: |
-          cd jabgui
-          ls
-          xcrun notarytool store-credentials "notarytool-profile" --apple-id "vorstand@jabref.org" --team-id "6792V39SK3" --password "${{ secrets.OSX_NOTARIZATION_APP_PWD }}"
-          xcrun notarytool submit build/distribution/JabRef-${{ needs.build.outputs.major }}.${{ needs.build.outputs.minor }}${{ matrix.suffix}}.dmg --keychain-profile "notarytool-profile" --wait
-          xcrun stapler staple build/distribution/JabRef-${{ needs.build.outputs.major }}.${{ needs.build.outputs.minor }}${{ matrix.suffix}}.dmg
+          for dir in jabgui jabkit; do
```

Comment:
this does not make sense, we do not build a pkg or dmg for jabkit

Reply:
Since the PR is a hot fix - I will revert

Let's discuss how the jabkit can be distributed on macOS - if it is not `jpackage`.